packages/@lwc/engine-core/src/framework/base-lightning-element.ts

```diff
@@ -194,6 +195,7 @@ function BaseLightningElementConstructor(this: LightningElement): LightningEleme
     }
 
     const component = this;
+    setPrototypeOf(elm, bridge.prototype);
```

Comment:
this is the first operation done by the super constructor,  which should be sufficient.